packages/subgraph/src/packages/multisig/multisig.ts

```diff
@@ -60,9 +60,9 @@ export function _handleProposalCreated(
 
     // if minApproval is 1, the proposal is always executable
     if (parameters.minApprovals == 1) {
-      proposalEntity.executable = true;
+      proposalEntity.potentiallyExecutable = true;
```

Comment:
When you initialize the `proposalEntity` at the top of the method. Shoud you add a `proposalEntity.potentiallyExecutable = false;` to make sure is not null, like you did on the addresslist or is there something else that ensures that is not null?

Reply:
this one is not about the null,
it is True here bcoz minApproval is 1, as explained in the comment above.

Reply:
👍